pkg/edgeview/src/edge-view-init.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
 
+if [ -f "/config/remote_access_disabled" ]; then
+  echo "Remote access is disabled"
```

Comment:
Does this line make it to Kibana?

Reply:
I don't think so, but I have an idea.

Reply:
If not you can make zedagent read it and log based on that.

It is useful to have zedagent read it in any case since that way zedagent/parseconfig.go can return an error if debug.enable.ssh is set.

Reply:
logs written to kmgs are picked up by newlogd, so I'm writing to that. For ssh, I added some lines to not start the sshd but keep the container alive.